### PR TITLE
Upgrade cluster-api kubekins-e2e version (v20220829-d5650c79bf-1.24 -…

### PR DESCRIPTION
**Pull request by oscr**
Upgrading as part of https://github.com/kubernetes-sigs/cluster-api/issues/6661

/assign @sbueringer

### CODE REVIEW

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml [hunk 1]
```diff
@@ -275,7 +275,7 @@ presubmits:
     spec:
       serviceAccountName: prowjob-default-sa
       containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220829-d5650c79bf-1.24
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220829-d5650c79bf-1.25
         args:
           - runner.sh
           - "./scripts/ci-e2e.sh"
```
Comment (sbueringer) [inline config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml:281]: There is one more image below (pull-cluster-api-e2e-workload-upgrade-1-21-1-22-main)

Comment (sbueringer) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml:281]: @oscr Can you please also open a second PR to do the same in cluster-api-periodics-main-upgrades.yaml and cluster-api-periodics-main.yaml
(intentionally a separate PR so I can merge the second one after I know the presubmit jobs are working after the first one)

**Comment by sbueringer** [comment]
@oscr Btw you can open an org membership PR for the kubernetes org (as you're already a member of kubernetes-sigs)

xref: https://github.com/kubernetes/community/blob/master/community-membership.md#kubernetes-ecosystem

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml (hunk 1 above)
Comment (oscr) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml:281]: @sbueringer Great catch! I don't know how I missed it, and you where able to catch it. This pr is updated. I will fix a separate as requested.

**Comment by sbueringer** [comment]
Thx!
/lgtm
/approve

File: config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml (hunk 1 above)
Comment (sbueringer) [reply config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-main.yaml:281]: Perfect, thx! :)

**Pull request merged**